llvm/lib/Target/SPIRV/SPIRVInstructionSelector.cpp

```diff
@@ -1743,7 +1743,7 @@ bool SPIRVInstructionSelector::selectDot4AddPackedExpansion(
   assert(I.getOperand(4).isReg());
   MachineBasicBlock &BB = *I.getParent();
 
-  bool Result = false;
+  bool Result = true;
```

Comment:
The changes to result here are from review comments in the previous `i8` version of the pull requests, feedback. I had mistakenly replaced the `Result` in `selectIntegerDot` instead of here.

Reply:
Should this be part of it own PR since it is a bug fix?

Reply:
Good idea. It would be probably also be a good idea to audit the other uses of `Result |=` in the file, as I think there are other instances it was mistakenly used. I will create a new pr with those and this in it.